### ISSUE TITLE
Remove unethically trained diffusion models from model hub

### ISSUE THREAD

**Issue by lukovnikov** [huggingface/diffusers#1716]
@patrickvonplaten @patil-suraj @anton-l 
I wonder what your thoughts are on the arguments the artists are making on Artstation, Twitter and Instagram right now. The main point is that the artists didn't give permission for their work to be scraped and used to create an AI that will threaten their livelihood, without compensation. I've already seen multiple mentions pop up on social media where Huggingface checkpoints are being linked, which are even being advertised as trained on the work of well-known artists (without their consent obviously).

It would be nice if we and Huggingface take a well-informed position and set an example.

Below are a few links, but it's not hard to find if you just Google for it:

https://www.vice.com/en/article/ake9me/artists-are-revolt-against-ai-art-on-artstation
https://www.reddit.com/r/midjourney/comments/zlim7e/artstation_this_evening/

**Comment by lukovnikov** [comment]
so [here](https://www.instagram.com/p/CmMds1dM__Z/?utm_source=ig_web_copy_link) for example Lois Van Baarle (@loisvb on instagram) claims her images have been used in the datasets (however she didn't specify which models or datasets but I'm assuming stable diffusion), i'm quoting:

> ... Because my artwork is included in the datasets used to train these image generators without my consent. I get zero compensation for the use of my art, even though these image generators cost money to use, and are a commercial product...

So should she go ahead and flag stable diffusion models? Do we know what subset of LAION stable diffusion was trained on?

**Comment by Cyberes** [comment]
@lukovnikov I find it really absurd that someone would complain about this in a issue on a CLI tool for distributed training. I briefly scrolled through your profile and saw that you were interested in the CelebA-HQ-256 dataset. Well, there's something called the *Unauthorized Use of Identity* which is the "right control your name, likeness, voice, signature, or other personal identifying traits." It's pretty hypocritical of you to whine about "AI stealing art" when you're doing the exact same thing with people's identities.

What's odd is that you appear have considerable experience with AI technology yet your complaint reads like something an uninformed average Joe would write. Are you going for the woke points? Did you tweet how "woke" you are?

> I don't know much about copyright but do you think this is ok?

You can drop the act, we all know the intention of your vapid questions. I vote to close this issue before the shit-flinging begins.

**Comment by lukovnikov** [comment]
@Cyberes Well, that escalated quickly and I guess I see how the slinging started. I see you are a prolific generative promptist yourself.

I think this issue very much deserves to be here and expected constructive arguments rather than personal attacks. If we can argue why LAION is fair to use, it would be good to have on this project since we're distributing the models.

CelebA is for non-commercial purposes only while stable diffusion is being re-trained and sold (just this morning @dansuiart (very mild NSFW warning) announced selling his kit for ~50$ for example). Mid-journey charges money for usage. So obviously these scraped images are used to create products for commercial purposes. However, the original artists have never agreed to this and are not being paid for generating training data. At the very very least, the license should not allow commercial uses and even then, I wonder how ok it is. But as I said, I don't know much about copyright, but I completely understand the artists.

If you remember, a year ago, Github Copilot was in a similar situation, where there was risk of open-source code being used in commercial projects, which would go against license terms. And I think many open-source developers had similar feelings about their open-source work being monetized upon by Microsoft without any compensation.

> You can drop the act, we all know the intention of your vapid questions. I vote to close and lock this issue before the shit-flinging begins.

You must know more than me, like I said I expect a constructive discussion and change if needed.

**Comment by patrickvonplaten** [comment]
Thanks for the comments. Personally, I believe this question is a more related to a datasets library or directly the model repositories on the Hub where the weights reside.

I would be in favor of closing it since we as maintainers cannot and should not take decisions whether model weights should be taken down or not.

**Comment by averad** [comment]
@lukovnikov

```
It would be nice if you could flag such model repositories.
There is a button that says "Report this model"
If relevant the Hugging Face Hub will then act upon the report.
```

patrickvonplaten referred you to https://huggingface.co/ where you can use the Report this model button for each model you are concerned with.

**Comment by julien-c** [comment]
haven't seen this mentioned here BTW but "report this model" works: e.g. see this thread where a model repo was blocked because the artist didn't consent to his work being used to train a model

- https://huggingface.co/sd-concepts-library/dyoudim-style/discussions/2
- https://twitter.com/julien_c/status/1600764649859076096

**Comment by lukovnikov** [comment]
@averad thank you for the very comprehensive and extensive answer. Maybe it should be extended into a blog post. I definitely don't want to blame the tools and think it's exciting to develop this technology. Most artists don't have an issue with progress although they realize it will become even harder, especially for the smaller ones, to make income, and will overall squeeze their labour market even more. The issue most artists are having is that their images were trained on without asking for their consent and then used in commercial applications that already are competing or will compete with the artistic community's main sources of income.

Regarding the food for thought, it's a very interesting question but fortunately the issue here doesn't lie that deep, it's about a data grab for applications that clearly don't fall under fair use anymore. Like you implied, generative models will (?) only ever be as good as the data and I doubt it would be as commercially attractive if it wasn't trained on copyrighted material. So I don't understand how stable diffusion models and others are allowed to be commercially used.

So to be consistent with the example @julien-c posted, shouldn't we also take down all stable diffusion models trained on LAION(-linked) images (but again, it's not clear what subset it was really trained on, from what I could quickly find). Among many others, Lois Van Baarle clearly stated she did not consent and her images are (linked) in both LAION-500m and LAION-5B.

**Comment by averad** [comment]
@lukovnikov just for clarification, have you or the people you are representing used the "report this model" function on hugging face to voice your concerns? Also have have you or the people you are representing submitted a request to laion.ai to have images removed?

The reason I ask is because if a model has been taken down (Such as SD v1) or if the images were removed from the training set it would help your case.

**Comment by lukovnikov** [comment]
@averad I'm not really representing anyone but as a scientist, I find this situation wildly unfair. We pay annotators on mechanical turk more than what the artists received for generating the training examples (0$). I don't know much about copyright and current laws but it is very possible the laws aren't ready yet to do the right thing right now. 

But some food for thought is this: do you think it's transformative to run a Fourier transform on a bunch of Disney movie frames, adding some noise in frequency domain and then reselling the inverse transforms of a random weighted sum of the transformed frames? How is a trained model fundamentally different from that, considering the training objective is simply a reconstruction loss on the training data?

I reported [stabilityai/stable-diffusion-2](https://huggingface.co/stabilityai/stable-diffusion-2/discussions) on the hub and the only reply I got so far was "and?".

Personally, I think at least any model trained on LAION images should be restricted to research/education-only, if that's possible. But if you think of it as a Fourier transform of copyrighted material, I think it should not exist or be distributed at all. Regarding flagging images, don't you think it's better/fairer to have an opt-in strategy and train on public domain data (like apparently stabilityAI did for their music generator in order to avoid possible copyright issues)?

**Comment by averad** [comment]
https://en.wikipedia.org/wiki/Fan_art

**Comment by lukovnikov** [comment]
@averad 
From the fanart page: "The legal status of derivative fan made art in America may be tricky due to the vagaries of the United States [Copyright Act](https://en.wikipedia.org/wiki/Copyright_Act_of_1976). Generally, the right to reproduce and display pieces of artwork is controlled by the original author or artist under [17 U.S.C.](https://en.wikipedia.org/wiki/17_U.S.C.) § 106. Fan art using settings and characters from a previously created work could be considered a [derivative work](https://en.wikipedia.org/wiki/Derivative_work), which would place control of the copyright with the owner of that original work. Display and distribution of fan art that would be considered a derivative work would be unlawful.

However, [American copyright law](https://en.wikipedia.org/wiki/Copyright_law_of_the_United_States) allows for the production, display and distribution of derivative works if they fall under a [fair use](https://en.wikipedia.org/wiki/Fair_use) exemption, 17 U.S.C. § 107. A court would look at all relevant facts and circumstances to determine whether a particular use qualifies as fair use; a multi-pronged rubric for this decision involves evaluating the amount and substantiality of the original appropriated, the transformative nature of the derivative work, whether the derivative work was done for educational or noncommercial use, and the economic effect that the derivative work imposes on the copyright holder's ability to make and exploit their own derivative works. None of these factors is alone dispositive."

So I don't see how these links add to the discussion.

**Comment by averad** [comment]
What did laion.ai say when you contacted them?
